### PR TITLE
Clean and simplify POI panel structure

### PR DESCRIPTION
**Pull request by bbecquet**
## Description
Simplify stuff about the POI Panel, trying to reduce its complexity:
- regroup component files in the same directory
- use the `<Panel>` component, when it's not in compact mode (aka. "card")
- remove a lot of useless/specific CSS
- slightly improve some UX details in the process, like the close button on the mobile card which is consistently present and never collides with the action buttons anymore (fixing a part of QMAPS-1066)

## Why
- Improve style cohesion everywhere: now all panels use the standard `<Panel>` component, ensuring visual changes to it are applied everywhere
- Simplify code readability and maintenability

### CODE REVIEW

**Comment by amatissart** [comment]
On Chrome mobile the Poi panel is not correctly positioned

**Comment by bbecquet** [comment]
@amatissart :wave: 

> On Chrome mobile the Poi panel is not correctly positioned
> 

I think I fixed it. In any case I can't reproduce it on my device, but I'm interested in you testing in again :)
I rebased this PR on [the one](https://github.com/QwantResearch/erdapfel/pull/516) including panel DOM simplifications, so I'll let the **WIP** flag until the other is merged.

**Comment by bbecquet** [comment]
Rebase on master done, removing WIP and ready to review :)

File: src/panel/poi/PoiPanel.jsx [hunk 1]
```diff
@@ -202,94 +192,57 @@ export default class PoiPanel extends React.Component {
...
-                <span className="icon-icon_compass" />{_('Search around this place', 'poi')}
-              </h3>
-              <CategoryList />
-            </div>}
-            {isFromOSM(poi) && <OsmContribution poi={poi} />}
-          </div>
+      }
+      {!backAction && pagesjaunes &&
+        <div className="poi_panel__pj_logo_container">{pagesjaunes}</div>
+      }
```
Comment (amatissart) [inline src/panel/poi/PoiPanel.jsx]: This block should be rendered when accessing "pagesjaunes" POIs via their URL, but `backAction` is always defined...

Comment (bbecquet) [reply src/panel/poi/PoiPanel.jsx]: Thanks for your vigilance :) :policeman: 
I think I fixed this behavior.

**Pull request merged**